manifests/init.pp

```diff
@@ -257,6 +260,7 @@
   Integer $quartz_job_threadcount                               = $rundeck::params::quartz_job_threadcount,
   Rundeck::Loglevel $rd_loglevel                                = $rundeck::params::loglevel,
   Rundeck::Loglevel $rd_auditlevel                              = $rundeck::params::loglevel,
+  Boolean $rd_compress_log                                      = $rundeck::params::compress_log,
```

Comment:
since the default is always the same, please don't put it into params.pp. set it directly in the init.pp